doc/cryptocurrency-api-exchange-integration.md

```diff
@@ -74,6 +74,8 @@ JSON array of objects (one for each market) containing the following properties:
 - `id`: **Required** The exchange's ID of the market
 - `base`: **Required** The base currency of the market
 - `quote`: **Required** The quote currency of the market
+- `type`: **Required** The type the market. Valid values are `spot`, `future`, `option`, `index`, and `derivative` (discouraged).

```

Comment:
This seems like it could be a little unclear, like all the values are discouraged. Maybe something like:

`The type of the market. Valid values are spot, future, option, and index. If the market is a derivative but not an option or future, use "derivative"`.

Reply:
Yeah that's way better 👍